<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetmemDialog</class>
 <widget class="QDialog" name="SetmemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Flash Memory</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>251</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Device 4 for default internal memory</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>126</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="radioDev0">
      <property name="text">
       <string>Device 0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioDev1">
      <property name="text">
       <string>Device 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioDev2">
      <property name="text">
       <string>Device 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioDev3">
      <property name="text">
       <string>Device 3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioDev4">
      <property name="text">
       <string>Device 4</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
